pkg/registry/autoscaling/rest/storage_autoscaling.go

```diff
@@ -100,6 +102,7 @@ func (p RESTStorageProvider) v2beta2Storage(apiResourceConfigSource serverstorag
 	return storage, err
 }
 
-func (p RESTStorageProvider) GroupName() string {
+// GroupName is the group name use in this package
```

Comment:
How about "GroupName returns the canonical group name of the autoscaling group"?